index.bs

```diff
@@ -2581,6 +2581,38 @@ value explicitly. <em>block-size</em> is defined to be 128
 sample-frames which corresponds to roughly 3ms at a sample-rate of
 44.1 kHz.
 
+<h4 id="AudioNode-tail">AudioNode tail-time</h4>
+
+An {{AudioNode}} can have a <dfn>tail-time</dfn> that is greater than zero. This
+means that even when the {{AudioNode}} is fed silence, non-silent audio can be
+output.
+
+<h4 id="AudioNode-actively-processing">AudioNode lifetime</h4>
+
+{{AudioNode}} can be <dfn id="actively-processing">actively processing</dfn>
+during a <a>render quantum</a>.
+
+- An {{AudioScheduledSourceNode}} is [=actively processing=] when and only when it
+	is playing for at least part of the current rendering quantum.
+- A {{DelayNode}} in a cycle is actively processing only when the absolute value
+	of any output sample for the current [=render quantum=] is greater than or equal
+	to \( 2^{-126} \).
+- A {{ScriptProcessorNode}} is [=actively processing=] when its input or output is
+	connected.
+- An {{AudioWorkletNode}} is [=actively processing=] when its
+	{{AudioWorkletProcessor}}'s {{[[callable process]]}} is <code>true</code> and
+	either its [=active source=] flag is <code>true</code> or any {{AudioNode}}
+	connected to one of its inputs is [=actively processing=].
+- All other {{AudioNode}}s start [=actively processing=] when any
+	{{AudioNode}} connected to one of its inputs is [=actively processing=], and
+	stops [=actively processing=] when the input that was received from other
+	[=actively processing=] {{AudioNode}} no longer affects the output.
```

Comment:
"no longer affects the output" seems too vague.  Not sure if we can say anything better, but we should try.  I'm pretty sure Chrome and Firefox differ on when biquad tails are finished.

Reply:
Maybe we should not say "no longer effects output" but say that the tail-time of the node has passed.

Also, I think we want to rephrase "the input that was received from other actively processing AudioNode".  If the input is actively processing, the current node is also processing, per the previous bullet item.  I think we need to say that if all inputs are not actively processing, the current node continues to process until the tail time has passed at which point the node is no longer actively processing.

Is that too complicated?

Reply:
Referring to the tail-time is fine with me if that's clearer.  See my comment on tail-time for an attempt to more clearly describe this there.  We need to be a little careful to avoid the two definitions becoming too cyclic.

"the input that was received from other" is no longer required if using tail-time.  Perhaps:

"When there are no longer any actively processing AudioNodes connected to its inputs, an AudioNode continues actively processing for the length of its tail-time.  The AudioNode stops actively processing when its tail-time expires before it again has any actively processing AudioNodes connected to its inputs."